browser/css/btns.css

```diff
@@ -170,6 +170,11 @@ button.has-img {
 	min-width: auto !important;
 }
 
+/* Thesaurus <- button */
+button#left.has-img > img {
```

Comment:
better to add ID of the dialog before to target only specific dialog ("left" is popular name probably)
But shouldn't break anything as icons inside buttons will be similar size.